secant/Features/BackupFlow/Views/RecoveryPhraseDisplayView.swift

```diff
@@ -52,28 +57,30 @@ struct RecoveryPhraseDisplayView: View {
                                         .bodyText()
                                 }
                             )
-                            .frame(height: 60)
+                                .frame(height: 60)
                         }
                         .padding()
                     } else {
                         Text("Oops no words")
                     }
                 }
-                .padding()
             }
+            .padding(.bottom, 20)
             .padding(.horizontal)
+            .padding(.top, 0)
+            .applyScreenBackground()
         }
-
+        
         // TODO: NavigationBar Style
```

Comment:
If the navigation bar is hidden now, do we need a TODO to style it?

Reply:
![image](https://user-images.githubusercontent.com/484008/151636545-e18bb059-2661-408e-9ece-9fbd343eecb2.png)
I understand this is a nav bar. Isn't is @geffenz

Reply:
Thanks, I'll do so. I never imagined that our compiler was so.... well.. I'll let the reader choose the adjective. 😅